Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -967,4 +980,455 @@ open class NavigationMapView: UIView {
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+        }
+
+        let stretchX = [ImageStretches(first: Float(32), second: Float(42))]
+        let stretchY = [ImageStretches(first: Float(26), second: Float(32))]
+        let imageContent = ImageContent(left: 32, top: 26, right: 47, bottom: 33)
+
+        // Right-hand pin
+        if let image =  UIImage(named: "AnnotationRightHanded", in: .mapboxNavigation, compatibleWith: nil) {
+            let regularAnnotationImage = image.tint(.intersectionAnnotationDefaultBackgroundColor)
+
+            style.setStyleImage(image: regularAnnotationImage,
+                                with: "AnnotationRightHanded",
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+
+            let highlightedAnnotationImage = image.tint(.intersectionAnnotationSelectedBackgroundColor)
+            style.setStyleImage(image: highlightedAnnotationImage,
+                                with: "AnnotationRightHanded-Highlighted",
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+        }
+
+        // Left-hand pin
+        if let image =  UIImage(named: "AnnotationLeftHanded", in: .mapboxNavigation, compatibleWith: nil) {
+            let regularAnnotationImage = image.tint(.intersectionAnnotationDefaultBackgroundColor)
+
+            style.setStyleImage(image: regularAnnotationImage,
+                                with: "AnnotationLeftHanded",
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+
+            let highlightedAnnotationImage = image.tint(.intersectionAnnotationSelectedBackgroundColor)
+            style.setStyleImage(image: highlightedAnnotationImage,
+                                with: "AnnotationLeftHanded-Highlighted",
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                scale: 2.0,
+                                imageContent: imageContent)
+        }
+    }
+
+    private func removeRouteAnnotationsLayerFromStyle() {
+        mapView.style.removeLayers([NavigationMapView.intersectionAnnotations])
+        _ = mapView.style.removeSource(for: NavigationMapView.intersectionAnnotations)
+    }
+
+    var annotationCache: AnnotationCache?
+    static let intersectionAnnotations = "intersectionAnnotations"
```

Comment:
This is redundant to `IdentifierString.intersectionAnnotationsLayer` above. This PR adds one layer identifier but applies styling according to another.

Reply:
Sorted out these identifiers when rebasing.